CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ else()
     if("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
         set(warnings "${warnings} -Wno-maybe-uninitialized")
     endif()
+
+    set(warnings "${warnings} -Wno-deprecated-copy -Wno-sign-compare")
```

Comment:
Isn't the no-sign-compare a righteous warning?

Reply:
ya, but sadly there is no way around this as beast I think spews this warning.